Assets/HoloToolkit/Input/Scripts/GameControllerManipulator.cs

```diff
@@ -7,7 +7,7 @@ namespace HoloToolkit.Unity.InputModule
 {
     /// <summary>
     /// Allows the user to place and rotate GameObjects using a game controller.
-    /// TODO This should be converted to an input source.
+    /// For a more completed game pad control script refer to GamepadInput.cs and MixedRealityTeleport.cs
```

Comment:
Why not mark obsolete if these classes should be used instead?
Besides this class is currently broken anyway.